app/themes/user-admin/e2e/user-edit/user-edit.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { UserAdminPage } from '../app.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User Edit Page', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.login();
+    });
+
+    it('should lead to User List page at click of the User List link', () => {
+        //Click UserList in Nav to get to UserList Page
+        let userListNav = browser.findElement(protractor.By.cssContainingText('nav a', 'User List'));
+        userListNav.click();
+    });
+
+    //this test should lead to the user created inside the user-create e2e test    
+    it('should lead to individual user edit page at click of EDIT', () => {
+
+        let edit = browser.findElement(protractor.By.css('tbody tr:first-child a'));
```

Comment:
I tried to do something about that. If it get specific, the protractor freaks out on me. That thing can be really fragile.

I'll make another attempt at it today.